mobile-app/app/contexts/FeatureFlagContext.tsx

```diff
@@ -102,7 +108,7 @@ export function FeatureFlagProvider (props: React.PropsWithChildren<any>): JSX.E
 
   return (
     <FeatureFlagContext.Provider value={context}>
-      {props.children}
+      {!isError && !isLoading && props.children}
```

Comment:
can we add this condition on the return <></> ?

Reply:
Updated!